packages/store/src/StoreCore.sol

```diff
@@ -40,7 +40,11 @@ library StoreCore {
    */
   function initialize() internal {
     // Register internal schema table
-    registerSchema(StoreCoreInternal.SCHEMA_TABLE, SchemaLib.encode(SchemaType.BYTES32));
+    registerSchema(
+      StoreCoreInternal.SCHEMA_TABLE,
+      SchemaLib.encode(SchemaType.BYTES32, SchemaType.BYTES32),
```

Comment:
I think the first BYTES32 is supposed to be padding for keys now.
If so this took me a while to understand despite knowing how this all works pretty well, a comment would be useful

Reply:
The first `BYTES32` is the `valueSchema`, the second BYTES32 is the `keySchema`. Will add a comment. Also wanna turn the SchemaTable into an autogenerated table at some point, but there might be circular dependencies with the Schema existing...

Reply:
> The first BYTES32 is the valueSchema, the second BYTES32 is the keySchema

right that makes more sense, it's not just padding. What's the UINT256 one then?

Reply:
That's the Schema table's single primary key, which has type UINT256 because it represents a tableId. Added a comment about that too.